### ISSUE TITLE
Getting an error, I've tried a number of things but I'm just a pleb when it comes to python etc.

### ISSUE THREAD

**Issue by myndxero** [dfaker/stable-diffusion-webui-cv2-external-masking-script#12]
This is the error:
cv2.error: OpenCV(4.6.0) D:\a\opencv-python\opencv-python\opencv\modules\highgui\src\window.cpp:1250: error: (-2:Unspecified error) The function is not implemented.

I did this during installation according to your readme:
py -m pip install opencv-python

Requirement already satisfied: opencv-python in c:\users\myndx\appdata\local\programs\python\python310\lib\site-packages (4.6.0.66)
Requirement already satisfied: numpy>=1.19.3 in c:\users\myndx\appdata\local\programs\python\python310\lib\site-packages (from opencv-python) (1.23.4)

In general, I am so bloody frustrated with inpainting because there is no simple reliable solution for me. WebUI's inpainting gives me some dumb error "right is less than left" bs and now external inpainting breaks, though I was using it all the time. Why is inpainting so difficult!? 

I appreciate any help offered, sorry for my frustrations, I'm not savvy with all technical stuff.

**Comment by myndxero** [comment]
:< ongoing issue, unable to use default inpainting ( left is less than right error ) unable to use external masking.

**Comment by dfaker** [comment]
Seems to indicate it's the gui-less version that got installed for you `opencv-python-headless`.
Try the sequence of:

```
py -m pip uninstall opencv-python-headless
py -m pip uninstall opencv-python
py -m pip install --upgrade opencv-python
```

**Comment by dfaker** [comment]
closing as duplicate of #13

**Comment by myndxero** [comment]
Sorry I don't always see these notifications right away. For me, there seems to have been some conflict between different extensions. I now just do all my masking in photoshop. It's a few extra steps but reliable.

I dunno if something was getting "left behind" in memory switching between extensions, particularly with inpainting and causing strange things to happen.

**Comment by dfaker** [comment]
Yes you're quite right, two versions of cv2, one with and one without gui, particularly the case that `d8ahazard/sd_dreambooth_extension` uses the headless version that conflicts with this extension.

**Comment by myndxero** [comment]
I don't think I chose to install dreambooth, just started being packaged by default. It's good to get to the bottom of it that!  Thank you for that. I don't know if you or the dreambooth extension have resolved this scenario, but there's no reason I wouldn't use your extension again for the simpler easier jobs, I did quite like it and saves time when I don't need to do something more complicated in photoshop.
